server/indexers/auction.ts

```diff
@@ -135,13 +139,27 @@ async function processEvent(connection: PoolClient, eventLog: AuctionHouseEventL
     await insertAuctionBid.run(
       {
         tx: eventLog.transactionHash,
+        index: eventLog.transactionIndex,
         auctionId: Number(nounId),
         walletAddress: sender.toLowerCase(),
         value: value.toString(),
         block: eventLog.blockNumber,
         extended: extended,
       },
-      connection
+      connection,
+    );
+    return;
+  }
+
+  if (eventLog.eventName === 'AuctionBidWithClientId') {
+    const { nounId, clientId, value } = (eventLog as AuctionBidWithClientIdEvent.Log).args;
+    await updateAuctionBidWithClientId.run(
+      {
+        auctionId: Number(nounId.toString()), // Noun id and auction id are the same
+        clientId: Number(clientId.toString()),
+        value: Number(value.toString()),
```

Comment:
you don't need `.toString()`s here, `Number(bigint)` works fine for these ranges of values